src/Query/MoreLikeThis.php

```diff
@@ -170,6 +170,8 @@ public function toArray(): array
             $doc = $array['more_like_this']['like'];
             $doc['doc'] = $array['more_like_this']['like']['_source'];
             unset($doc['_id'], $doc['_source']);
+            unset($doc['if_seq_no']);
```

Comment:
Trying to figure out how this change is related?

Reply:
The `if_seq_no` and `if_primary_term` parameters are now set by the fix and those parameters are not expected by the MoreLikeThis query. 
Find out because of a failing test.